src/main/java/com/netchill/api/moviedb/services/global/GlobalApiService.java

```diff
@@ -0,0 +1,37 @@
+package com.netchill.api.moviedb.services.global;
+
+import com.netchill.api.moviedb.TmdbApiClient;
+import com.netchill.api.moviedb.models.MovieDbPaginatedResponse;
+import com.netchill.api.moviedb.models.Production;
+import com.netchill.api.moviedb.services.serie.SerieApiService;
+import com.netchill.services.configuration.ConfigurationService;
+import okhttp3.MediaType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public class GlobalApiService {
+    private static final String LANGUAGE = "en-US";
+
+    private final GlobalApiRetrofit globalApi;
+    private final TmdbApiClient apiClient;
+    private final ConfigurationService configurationService;
+    @Inject
+    private GlobalApiService(TmdbApiClient apiClient, ConfigurationService configurationService) {
+        this.configurationService = configurationService;
+        this.apiClient = apiClient;
+        this.globalApi = this.apiClient.getRetrofitClient().create(GlobalApiRetrofit.class);
+    }
+
+    public MovieDbPaginatedResponse<Production> getTrending(@Nullable Integer page) {
+        return this.apiClient.executeRequest(globalApi.getTrending(
+                this.configurationService.getMovieDbApiKey(),
+                LANGUAGE,
+                page
+        ));
+    }
```

Comment:
ce serait bien d'avoir une surcharge pour cette méthode :

```suggestion
    public MovieDbPaginatedResponse<Production> getTrending() {
            return this.getTrending(null);
    }
    
    public MovieDbPaginatedResponse<Production> getTrending(@Nullable Integer page) {
        return this.apiClient.executeRequest(globalApi.getTrending(
                this.configurationService.getMovieDbApiKey(),
                LANGUAGE,
                page
        ));
    }
```